tests-functional/tests/test_init_status_app.py

```diff
@@ -41,49 +44,45 @@ def test_init_app(self):
         )
 
 
-@pytest.mark.rpc
-@pytest.mark.skip("waiting for status-backend to be executed on the same host/container")
-class TestInitializeLogging:
-
-    @pytest.mark.init
-    def test_init_logging(self, tmp_path):
-        self.check_logs(tmp_path, log_enabled=True, api_logging_enabled=True)
-
-    @pytest.mark.init
-    def test_no_logging(self, tmp_path):
-        self.check_logs(tmp_path, log_enabled=False, api_logging_enabled=False)
-
-    def assert_file_first_line(self, path, pattern: str, expected: bool):
-        assert os.path.exists(path) == expected
-        if not expected:
-            return
-        with open(path) as file:
-            line = file.readline()
-            line_found = line.find(pattern) >= 0
-            assert line_found == expected
-
-    def check_logs(self, path, log_enabled: bool, api_logging_enabled: bool):
-        data_dir = path / "data"
-        logs_dir = path / "logs"
-
-        data_dir.mkdir()
-        logs_dir.mkdir()
-
-        backend = StatusBackend()
-        backend.api_valid_request(
-            "InitializeApplication",
-            {
-                "dataDir": str(data_dir),
-                "logDir": str(logs_dir),
-                "logEnabled": log_enabled,
-                "apiLoggingEnabled": api_logging_enabled,
-            },
-        )
+def assert_file_first_line(path, pattern: str, expected: bool):
+    if not expected:
+        assert path is None
+        return
+    assert os.path.exists(path)
+    with open(path) as file:
+        line = file.readline()
+        line_found = line.find(pattern) >= 0
+        assert line_found == expected
 
-        self.assert_file_first_line(logs_dir / "geth.log", pattern="logging initialised", expected=log_enabled)
-
-        self.assert_file_first_line(
-            logs_dir / "api.log",
-            pattern='"method": "InitializeApplication"',
-            expected=api_logging_enabled,
-        )
+@pytest.mark.rpc
+@pytest.mark.init
+@pytest.mark.parametrize("log_enabled,api_logging_enabled", [(True, True), (False, False)])
+def test_check_logs(log_enabled: bool, api_logging_enabled: bool):
+    data_dir = os.path.join(USER_DIR, "data")
```

Comment:
`USER_DIR` seems somewhat ambiguous. It was unclear to me at first whether it refers to a path that should be set in the context of the host's environment or the container's (status-backend) environment.

Reply:
Well, it's the way to work with both docker and local environments.
I agree that it's unobvious, but I guess it's out of the scope of this PR. And I don't really have better ideas how to implement/name this.